parts/k8s/kubernetesmasterresourcesvmss.t

```diff
@@ -13,8 +13,8 @@
     "name": "[variables('clusterKeyVaultName')]",
     "apiVersion": "[variables('apiVersionKeyVault')]",
     "location": "[variables('location')]",
-    {{ if UseManagedIdentity}}
-    "dependsOn": 
+    {{if UseManagedIdentity}}
+    "dependsOn":
```

Comment:
I may have missed this somewhere, but I see a dependson for 
`"[concat('Microsoft.Authorization/roleAssignments/',guid(concat(variables('userAssignedID'), 'roleAssignment', resourceGroup().id)))]"` in kubernetesmasterresources.t

Are these dependencies also needed for kubernetesmasterresourcesvmss.t?
https://github.com/Azure/acs-engine/blob/master/parts/k8s/kubernetesmasterresourcesvmss.t#L264

Reply:
The depends on for system assigned identity were not present there, hence kept the new changes also similar to the existing one.